client/src/App.js

```diff
@@ -1,14 +1,18 @@
 import React, { Fragment } from "react";
+import { Router } from "@reach/router";
 import { createGlobalStyle } from "styled-components";
-
+import routes from "Constants/routes";
```

Comment:
I think Michaela mentioned this before, but you can expose this through an index.js and destructure it that way.

it would look like:

import { Routes } form "Constants";